.github/pull_request_template.md

```diff
@@ -2,4 +2,4 @@ Please ensure this checklist is followed for any pull requests for this repo. Th
 * [ ] Relevant documentation for this PR has to be completed and reviewed by @mdr0id before the PR can be merged
```

Comment:
This is something that we have never enforced, and honestly it doesn't seem like a good idea to bottleneck every PR on one individual. Instead, I think that we should have a requirement that the changelog has been updated.